herd/tests/instructions/AArch64.mixed/A161.litmus

```diff
@@ -0,0 +1,14 @@
+AArch64 A161
+(* Test load exclusive; Store exclusive, symbolic location *)
+(* Equivalent to A43 but testing mixed mode *)
+
+Variant=mixed
+
+ { 0:X4=z; uint64_t z=2; }
```

Comment:
By contrast with A157 and A159, one need not declare the type of register 0:X0. This is because the register is not printed. Register 0:X3 is printer but the default type `int` is correct.